src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const sourceObject = {};
+  const declarations = sourceString.split(';');
+
+  declarations.forEach((declaration) => {
+    const [property, value] = declaration.split(':');
+
+    if (property && value) {
+      const trimmedProperty = property.trim();
+      const trimmedValue = value.trim();
 
+      sourceObject[trimmedProperty] = trimmedValue;
+    }
```

Comment:
The check for 'property' and 'value' is good, but it does not account for the possibility of extra whitespace around the colon, which could result in an empty property or value after trimming. It would be safer to trim before checking for truthiness.